packages/back-end/src/scim/users/listUsers.ts

```diff
@@ -51,12 +51,12 @@ export async function listUsers(
 
   return res.status(200).json({
     schemas: ["urn:ietf:params:scim:api:messages:2.0:ListResponse"],
-    totalResults: filteredUsers.length,
+    totalResults: resources.length,
```

Comment:
I think the reason this was `filteredUsers` instead of `resources` is that `totalResults` is supposed to be the number of results regardless of the `count` or `startIndex` requested, but I have to look at the spec for SCIM to make sure

Reply:
I think the above is true so I think in this case `totalResults` should be `filteredUsers.length`

See https://datatracker.ietf.org/doc/html/rfc7644#section-3.4.2